programs/stake/src/points.rs

```diff
@@ -1,15 +1,7 @@
 //! Information about points calculation based on stake state.
 //! Used by `solana-runtime`.
 
-use {
-    solana_clock::Epoch,
-    solana_instruction::error::InstructionError,
-    solana_pubkey::Pubkey,
-    solana_stake_interface::state::{Delegation, Stake, StakeStateV2},
-    solana_sysvar::stake_history::StakeHistory,
-    solana_vote_interface::state::VoteState,
-    std::cmp::Ordering,
-};
+use {solana_pubkey::Pubkey, solana_stake_interface::state::Delegation};
 
```

Comment:
Is this "PointValue" still used?

Reply:
Not used by us anymore but I kept it there in case anyone downstream is using it and deprecated the entire `points` module